resources/console/profile-evaluation.yaml

```diff
@@ -0,0 +1,14 @@
+web:
+  deployment:
+    resources:
+      requests:
+        memory: 16Mi
+      limits:
+        memory: 32Mi
+backend: 
+  deployment:
+    resources:
+      requests:
+        memory: 32Mi
```

Comment:
On two different clusters I checked, the pod from this deployment consumes 104Mi and 116Mi at a `standby` state. I think we should request a value closer to these numbers.

Reply:
Here's the 12-hours `weekly` cluster summary from Grafana:
![image](https://user-images.githubusercontent.com/22825433/100615108-682d5d80-3317-11eb-898c-551087b6a865.png)